code-reviews/process-guidance/README.md

```diff
@@ -6,11 +6,21 @@ Code reviews should be part of the software engineering team process regardless
 
 To ensure that the code review process is healthy and meets the goals stated above, consider following these guidelines:
 
+- Establish a [service-level agreement (SLA)](https://en.wikipedia.org/wiki/Service-level_agreement) for code reviews and add it to your teams working agreement.
```

Comment:
Perhaps we could also add the suggestion of having a calendar invite that recurs everyday to remind people to take time to review PRs, that's one thing my team does and seems to work well